src/com/sun/ts/tests/servlet/compat/LeadingSlash/WithoutLeadingSlash/servlet_compat_LeadingSlash_Without_web.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<!DOCTYPE web-app PUBLIC '-//Sun Microsystems, Inc.//DTD Web Application 2.2//EN' 'http://java.sun.com/j2ee/dtds/web-app_2_2.dtd'>
 <!--
```

Comment:
We should revert the change to this file since it is invalid for this test.

Reply:
reverted this change to previous content since the test is excluded and cannot be updated to a recent schema as the test is only valid against web-app_2_2.dtd